### PR TITLE
Make close button icon white in error-type Infobars

### LINKED ISSUE #380: Infobar close icon always dark
It looks like the infobar close button is always dark, even on the error class. Instead, it should match the text color.

screenshot from 2018-08-14 10-56-49

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/62138467-infobar-close-icon-always-dark?utm_campaign=plugin&utm_content=tracker%2F45189256&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F45189256&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by lainsce**
When using the error class in infobars, makes the close button white instead of dark.
Fixes #380.

Looks like this:
Old =
old-error

New =
new-error

### CODE REVIEW

**Comment by lainsce** [comment]
Will fix :+1:

**Comment by lainsce** [comment]
Fixed~

File: elementary/gtk-3.0/gtk-widgets.css [hunk 1]
```diff
@@ -3560,7 +3560,8 @@ infobar.error revealer > box {
     -gtk-icon-palette: error #fff;
 }
 
-infobar.error label {
+infobar.error label,
+infobar.error.close {
```
Comment (cassidyjames) [inline elementary/gtk-3.0/gtk-widgets.css]: missing whitespace

**Comment by lainsce** [comment]
I should override `-gtk-icon-shadow` to "transparent", right?

**Comment by danrabbit** [comment]
@lainsce I would match to the text shadow

**Comment by lainsce** [comment]
> @lainsce I would match to the text shadow

Oh ok. Switching to text shadow then.

File: elementary/gtk-3.0/gtk-widgets.css [hunk 2]
```diff
@@ -3568,9 +3568,11 @@ infobar.error revealer > box {
     -gtk-icon-palette: error #fff;
 }
 
-infobar.error label {
+infobar.error label,
+infobar.error .close {
     color: #fff;
     text-shadow: 0 1px 1px @error_color;
+    -gtk-icon-shadow: 0 1px 1px alpha (shade (@error_color, 1.4), 0.4);
```
Comment (danrabbit) [inline elementary/gtk-3.0/gtk-widgets.css]: This should match the text shadow that is right above it and just be `@error_color`

**Pull request merged**